src/components/CardNewTeam.vue

```diff
@@ -72,9 +73,17 @@ export default {
               })
             }
           }
+        }).onCancel(async () => {
+          this.prompt = true
         })
       }
     }
   }
 }
 </script>
+
+<style>
+.q-pa-md {
+  text-align: center;
```

Comment:
remove these lines and add [text-center](https://quasar.dev/style/typography) to the classes of where you want the text to be centred

Reply:
the CSS is to center the button on the page